fastos/src/vespa/fastos/timestamp.h

```diff
@@ -61,6 +61,12 @@ class ClockSystem
     static int64_t now();
 };
 
+class ClockSteady
+{
+public:
+    static int64_t now();
```

Comment:
Would be good if the function name made it obvious what the unit was for the returned value, e.g. `now_nanos`

Reply:
I am now returning a TimeStamp so you have it all.